src/main/scala/it/pagopa/interop/authorizationprocess/api/impl/ClientApiServiceImpl.scala

```diff
@@ -613,4 +613,55 @@ final case class ClientApiServiceImpl(
 
   }
 
+  private def getRelationship(userId: UUID, consumerId: UUID, roles: Seq[String] = Seq(SECURITY_ROLE))(implicit
```

Comment:
Fine by me moving the role filter to parameters, but I would suggest to not use a default value
```suggestion
  private def getRelationship(userId: UUID, consumerId: UUID, roles: Seq[String])(implicit
```